app/models/edition.rb

```diff
@@ -32,6 +32,8 @@ class Edition
   field :change_note,          type: String
   field :review_requested_at,  type: DateTime
 
+  field :content_id,           type: String
```

Comment:
I don't think there's any need to have the content_id duplicated on the Edition. Every Edition should be tied to an Artefact and use that content_id?